docs/version-1.9/modules/en/pages/installation-setup/installation/install-using-argocd.adoc

```diff
@@ -1,14 +1,19 @@
-= Install {longhorn-product-name} Using ArgoCD
+= Install {longhorn-product-name} Using Argo CD
 :current-version: {page-component-version}
 
 == Prerequisites
 
+[NOTE]
+====
+{longhorn-product-name} installation requires authenticated access to your Kubernetes. For more details, check https://docs.apps.rancher.io/get-started/authentication/#kubernetes[authentication guide].
```

Comment:
The statement is wrong. It should be below. Please change others accordingly.

> {longhorn-product-name} installation requires authenticated access to Application Collection. For more details, check https://docs.apps.rancher.io/get-started/authentication[authentication guide].


P.S. JFYI.  Actually, users can use the following ways for installation and upgrade. 
- https://docs.apps.rancher.io/get-started/authentication/#helm
- https://docs.apps.rancher.io/get-started/authentication/#kubernetes